hdf5-sys/build.rs

```diff
@@ -204,6 +205,7 @@ impl Header {
                 hdr.have_threadsafe = value > 0;
             } else if name == "H5_HAVE_FILTER_DEFLATE" {
                 hdr.have_zlib = value > 0;
+                hdr.have_filter_deflate = value > 0;
```

Comment:
Ok, I'm somewhat confused, in HDF5 terminology isn't "deflate" = "zlib"?

Reply:
They are the mostly the same and used interchangeably. The compile definition is `HDF5_ENABLE_Z_LIB_SUPPORT` but the flag in `H5pubconf.h` is `H5_HAVE_FILTER_DEFLATE`.

Reply:
I don't mind having just `have_zlib` or `have_filter_deflate`

Reply:
Hm, just a single `have_filter_deflate` is probably more correct since that's the flag we're checking in `H5pubconf`?